src/main/drivers/rx/rx_sx127x.h

```diff
@@ -0,0 +1,338 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Based on https://github.com/ExpressLRS/ExpressLRS
+ * Thanks to AlessandroAU, original creator of the ExpressLRS project.
+ */
+
+#pragma once
+
+//SX127x_ModulationModes
+typedef enum {
+    SX127x_OPMODE_FSK_OOK = 0x00,
+    SX127x_OPMODE_LORA = 0x80,
+    SX127X_ACCESS_SHARED_REG_OFF = 0x00,
+    SX127X_ACCESS_SHARED_REG_ON = 0x40,
+} sx127x_modulation_mode_e;
+
+//SX127x_RadioOPmodes
+typedef enum {
+    SX127x_OPMODE_SLEEP = 0x00,
+    SX127x_OPMODE_STANDBY = 0x01,
+    SX127x_OPMODE_FSTX = 0x02,
+    SX127x_OPMODE_TX = 0x03,
+    SX127x_OPMODE_FSRX = 0x04,
+    SX127x_OPMODE_RXCONTINUOUS = 0x05,
+    SX127x_OPMODE_RXSINGLE = 0x06,
+    SX127x_OPMODE_CAD = 0x07,
+} sx127x_radio_op_mode_e;
+
+//SX127x_Bandwidth
+typedef enum {
+    SX127x_BW_7_80_KHZ = 0x00,
+    SX127x_BW_10_40_KHZ = 0x10,
+    SX127x_BW_15_60_KHZ = 0x20,
+    SX127x_BW_20_80_KHZ = 0x30,
+    SX127x_BW_31_25_KHZ = 0x40,
+    SX127x_BW_41_70_KHZ = 0x50,
+    SX127x_BW_62_50_KHZ = 0x60,
+    SX127x_BW_125_00_KHZ = 0x70,
+    SX127x_BW_250_00_KHZ = 0x80,
+    SX127x_BW_500_00_KHZ = 0x90,
+} sx127x_bandwidth_e;
+
+//SX127x_SpreadingFactor
+typedef enum {
+    SX127x_SF_6 = 0x60,
+    SX127x_SF_7 = 0x70,
+    SX127x_SF_8 = 0x80,
+    SX127x_SF_9 = 0x90,
+    SX127x_SF_10 = 0xA0,
+    SX127x_SF_11 = 0xB0,
+    SX127x_SF_12 = 0xC0,
+} sx127x_spreading_factor_e;
+
+//SX127x_CodingRate
+typedef enum {
+    SX127x_CR_4_5 = 0x02,
+    SX127x_CR_4_6 = 0x04,
+    SX127x_CR_4_7 = 0x06,
+    SX127x_CR_4_8 = 0x08,
+} sx127x_coding_rate_e;
```

Comment:
Still a lot of snake_case here and in rx_sx127x.c

Reply:
Sorry I was looking at one file at a time, too many changes. Fixed.